cli/src/commands/env.ts

```diff
@@ -73,9 +73,9 @@ export default class Env extends Command {
       description: 'list all variables',
       default: false,
     }),
-    'include-values': flags.boolean({
-      char: 'i',
-      description: 'include values of all variables with --list',
+    'hide-values': flags.boolean({
+      char: 'x',
```

Comment:
Is there a mnemonic for `x`?

Reply:
"x out those values" is the best I could do.  `-h` doesn't work because I expect it to be `--help`.